variables.tf

```diff
@@ -20,6 +33,7 @@ variable "region" {
 variable "prefix" {
   description = "The prefix that you would like to append to your resources. Explicitly set to null if you do not wish to use a prefix."
```

Comment:
Just noticed the word "append" here would indicate the value is added to the end of the resource name, which is not true. Suggest rewording to: "The value that you would like to prefix to the name of the resources provisioned by this module. Explicitly set to null if you do not wish to use a prefix."